sqlmesh/core/model/definition.py

```diff
@@ -751,31 +762,61 @@ def seed_path(self) -> Path:
             return self._path.parent / seed_path
         return seed_path
 
+    @property
+    def dehydrated(self) -> SeedModel:
+        """Creates a dehydrated copy of this model.
+
+        The dehydrated seed model will not contain the seed content, but will contain
+        the column hashes. This is useful for comparing two seed models without
+        having to read the seed content from disk.
+
+        Returns:
+            A dehydrated copy of this model.
+        """
+        if not self.is_hydrated:
+            return self
+        return self.copy(
+            update={
+                "seed": Seed(content=""),
+                "is_hydrated": False,
+                "column_hashes_": self.column_hashes,
+            }
+        )
+
+    def hydrated(self, content: str) -> SeedModel:
```

Comment:
these should be verbs, 

def hyrdate() and def dehydrate

Reply:
Not in this case since we don't mutate the instance, but return a new *hydrated* and *dehydrated* instance instead. So the full name of these methods should've been `create_hydrated` and `create_dehydrated` but I decided to shorten it in the best tradition of our naming strategy.

Reply:
it doesn't have to be a mutation to be a verb, like get_x() is not mutation

Reply:
Please also note that `dehydrated` is a property and will now become a method.

Reply:
sounds good